taxi/src/main/java/com/epam/training/taxi/AccountDTOImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.epam.training.taxi;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AccountDTOImpl implements AccountDTO{
+
+    public static final String PATH_TO_INPUT_CSV = "src/main/resources/accounts.csv";
+    public static final String PATH_TO_OUT_CSV = "./out.csv";
+
+    private List<Account> accounts;
+
+    @Override
+    public List<Account> readFromCSV(String path) throws IOException {
+        accounts = new ArrayList<>();
+        Files.lines(Path.of(path))
+                .map(line -> line.split(","))
+                .forEach(m -> accounts.add(
+                        new Account(Long.parseLong(m[0]),m[1],Double.parseDouble(m[2]))
+                        ));
+        return accounts;
+    }
+
+    @Override
+    public void writeToCSV(String path,String text) throws IOException {
+        FileWriter fw = new FileWriter(path,true);
+        fw.write(text + System.lineSeparator());
+        fw.close();
```

Comment:
I would like to suggest to use the try-with-resources here, which can close resources automatically which implement the AutoCloseable interface. In this case, there is no need to close the corresponding resource explicitly.
Please visit the following page for getting more information: https://www.baeldung.com/java-try-with-resources